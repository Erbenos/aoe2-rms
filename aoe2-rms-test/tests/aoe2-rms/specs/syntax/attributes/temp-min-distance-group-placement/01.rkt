#lang aoe2-rms
; Scatter neutral gold evenly across the map.

<OBJECTS-GENERATION>
(create-object 'GOLD
               (number-of-objects 9320)
               (number-of-groups 4)
               (set-gaia-object-only)
               (set-tight-grouping)
               (min-distance-group-placement 4)
               (temp-min-distance-group-placement 46))
