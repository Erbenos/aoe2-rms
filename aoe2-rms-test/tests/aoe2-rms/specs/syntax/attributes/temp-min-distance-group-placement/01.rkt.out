<OBJECTS_GENERATION>
create_object GOLD {
  number_of_objects 9320
  number_of_groups 4
  set_gaia_object_only
  set_tight_grouping
  min_distance_group_placement 4
  temp_min_distance_group_placement 46
}
